src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Layers/RasterColormapRenderer/readme.md

```diff
@@ -0,0 +1,29 @@
+# Colormap renderer
+
+Demonstrates how to use a colormap renderer on raster layer.
+Colormap renderers can be used to replace values on a raster layer with a color based on the original brightness value.
+
+![](RasterColorMapRenderer.jpg)
+
+## How it works
+To apply a `ColormapRenderer` to a `RasterLayer`:
+
+- Create a `Raster` from a raster file.
+- Create a `RasterLayer` from the raster.
+- Create an `IEnumerable<Color>` and use it as a colormap: colors at the beginning of the list replace the darkest values in the raster and colors at the end of the list replaced the brightest values of the raster.
+- Create a colormap renderer with the colormap and apply it to the raster layer with `rasterLayer.Renderer = colormapRenderer`.
+
+## Features
+- `Map`
+- `Basemap`
+- `ColormapRenderer`
+- `MapView`
+- `Raster`
+- `RasterLayer`
```

Comment:
There are a few aspects of this readme that appear to be out of date. I recommend copying from the latest version to correct some formatting issues (super picky markdown stuff)